src/components/AllArticlesComp/AllArticles.jsx

```diff
@@ -0,0 +1,71 @@
+import Article from '../ArticleComp/Article';
+import React from 'react';
+import ListPagination from '../ListPaginationComp/ListPagination';
```

Comment:
please make sure to use the absolute import here and every where in the app
```suggestion
import ListPagination from 'components/ListPagination/ListPagination';
```